controllers/api/forum-routes.js

```diff
@@ -5,71 +5,75 @@ const withAuth = require('../../utils/auth');
 
 //gets all forum posts
 router.get('/', async (req, res) => {
```

Comment:
```suggestion
router.get('/', withAuth, async (req, res) => {
```
Without "withAuth" middleware, we aren't checking to see if the user is logged in before accessing the forums.